src/components/Features.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Featurescomponent
+ * This is the features and some information
+ */
+
+/* Imports -------------------------------------------------------------------*/
+
+import Inferno from 'inferno';
+import Component from 'inferno-component';
+
+/* Local variables -----------------------------------------------------------*/
+
+const version = '^3.0.0';
+
+/* Component -----------------------------------------------------------------*/
+
+class VersionComponent extends Component {
+	constructor(props) {
+		super(props);
+	}
+
+	showVersion() {
```

Comment:
the following code was removed:
constructor(props) {
		super(props);
	}

	showVersion() {
		alert(`The version is: ${ version }!`);
	}